src/base/App.js

```diff
@@ -5,6 +5,7 @@ import { useDispatch } from 'react-redux';
 //Components
 import Modal from './../components/Modal/Modal';
 import Navbar from './../components/Navbar/Navbar';
+import NavMobile from './../components/NavMobile/NavMobile';
```

Comment:
since youre working in this file can you update these relative paths to be `components/blah/blah`